src/components/bootstrap/bootstrap.scss

```diff
@@ -0,0 +1,5 @@
+@import url('~bootstrap/dist/css/bootstrap.css');
```

Comment:
This works, but alternatively, with webpack you should be able to import this directly into the `bootstrap` component:
`import 'bootstrap/dist/css/bootstrap.css';`

Reply:
neat!